evap/static/ts/src/notebook.ts

```diff
@@ -86,3 +84,10 @@ export class NotebookLogic {
         this.formLogic.attach();
     };
 }
+
+
+export function initNotebook(notebookSelector: string, noteBookFormSelector: string, evapContentSelector: string, localStorageKey: string, collapseNotebookButtonSelector: string): void {
+    if (document.querySelector(notebookSelector) && document.querySelector(noteBookFormSelector) && document.querySelector(evapContentSelector) && localStorageKey && collapseNotebookButtonSelector) {
```

Comment:
No `if` required here after we've moved the JS into notebook.html: If someone calls this functions with invalid selectors/arguments, hard error to the console to make it debuggable.

After that, I think there is no point in the function anymore and we can go back to just using the class / constructor directly.